src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalInstantiationCheckTest.java

```diff
@@ -51,25 +52,27 @@ protected String getNonCompilablePath(String filename) throws IOException {
     }
 
     @Test
-    public void testIt() throws Exception {
+    public void testDefault() throws Exception {
         final DefaultConfiguration checkConfig =
             createCheckConfig(IllegalInstantiationCheck.class);
         checkConfig.addAttribute(
             "classes",
             "java.lang.Boolean,"
-                + "com.puppycrawl.tools.checkstyle.checks.coding.InputModifier,"
+                + "com.puppycrawl.tools.checkstyle.checks.coding."
+                + "illegalinstantiation.InputModifier,"
                 + "java.io.File,"
                 + "java.awt.Color");
         final String[] expected = {
             "19:21: " + getCheckMessage(MSG_KEY, "java.lang.Boolean"),
             "24:21: " + getCheckMessage(MSG_KEY, "java.lang.Boolean"),
             "31:16: " + getCheckMessage(MSG_KEY, "java.lang.Boolean"),
             "38:21: " + getCheckMessage(MSG_KEY,
-                "com.puppycrawl.tools.checkstyle.checks.coding.InputModifier"),
+                "com.puppycrawl.tools.checkstyle.checks.coding."
+                + "illegalinstantiation.InputModifier"),
```

Comment:
@romani can decide then, but we didn't want to have violations change just so we didn't have to deeply review the changes to the input file and the behavior of the check to make sure it is not because of some other change.